src/module/node/reducer.ts

```diff
@@ -1,36 +1,26 @@
 import { getType } from 'typesafe-actions';
 import { INodesState } from '@/type/types';
-import { resizeNode, nodeActionTypes } from './actions';
+import uniqid from 'uniqid';
```

Comment:
never used uniqid before, how does it differ from uuid?

Reply:
I don't think there is much difference just generate unique ID - can use uuid if you prefer longer term ?